Added OpenShot CheatSheet

tagawa opened the pull request:
**What does your Instant Answer do?**
Cheat sheet for the OpenShot video editing software (Linux)

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Keyboard shortcut lookup.

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.openshotusers.com/help/1.4/en/ar01s05.html

**Why did you choose this data source?**
It's the definitive source.

**Are there any other alternative (better) data sources?**
Not that I'm aware of.

**What are some example queries that trigger this Instant Answer?**
openshot cheatsheet
openshot cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Linux users, video editing people

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No, but I asked for approval first.

**Which existing Instant Answers will this one supercede/overlap with?**
None.

**Are you having any problems? Do you need our help with anything?**
Nope, all good.

**Where did you hear about DuckDuckHack? (For first time contributors)**
A colleague.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
![openshot-cheat-sheet](https://cloud.githubusercontent.com/assets/94173/8276664/1dfc9098-18f4-11e5-84fb-12b4705da487.png)

## Checklist
Please place an 'X' where appropriate.

```
- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [x] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Linux Ubuntu 14.04
        - [x] Chromium
        - [x] Firefox
        - [x] Opera

    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

------
IA Page: https://duck.co/ia/view/open_shot_cheat_sheet

moollaza commented on share/goodie/cheat_sheets/openshot.json:
```diff
@@ -0,0 +1,111 @@
+    "section_order": [
+        "Editing"
+    ],
+    "sections": {
```
Any chance we could break this up into more sections? Would make it look much better in the full, expanded view

moollaza commented on share/goodie/cheat_sheets/openshot.json:
```diff
@@ -0,0 +1,111 @@
+{
+    "id": "openshot_cheat_sheet",
```
please change to `open_shot_cheat_sheet`, to match the IA Page: https://duck.co/ia/view/open_shot_cheat_sheet

tagawa commented:
![openshot-cheat-sheet](https://cloud.githubusercontent.com/assets/94173/8343484/938b0eb0-1a8a-11e5-8877-7c7d63ec37d2.png)

Thanks for the feedback @moollaza. Creating sections looks much better.

moollaza commented on share/goodie/cheat_sheets/openshot.json:
```diff
@@ -0,0 +1,115 @@
+    ],
+    "sections": {
+        "General": [{
+            "key": "CTRL + N",
```
@tagawa We actually *just* updated the syntax and we've decided to ditch the `+` between keys.

Can you update these like so:

```
// old
Ctrl + O

//new
[Ctrl] [O]
```

This will wrap each key (signified by wrapping in `[]`) in a `<code>` block and will make it look **much** cooler.

tagawa replied:
Well if it'll also make *me* much cooler then sure!

tagawa commented:
With key codes now using square brackets:

![openshot-cheatsheet](https://cloud.githubusercontent.com/assets/94173/8641742/d0760a36-2954-11e5-9c7f-0e87bf2226f4.png)

moollaza commented on share/goodie/cheat_sheets/openshot.json:
```diff
@@ -0,0 +1,115 @@
+            "val": "Quit the program"
+        }],
+        "Playback": [{
+            "key": "[J]",
```
@tagawa sorry about the confusion! The syntax is a bit funny, but single strings, without spaces, **don't** need to be wrapped in `[]` in order to be put into a codeblock. So if the `"key"` is a single letter, or word and has no spaces ` ` you shouldn't wrap it in `[]` otherwise the `[` and `]` will be taken literally and as you can see, they end up in the  actual cheat sheet!

tagawa commented:
Thanks for the alert @moollaza &mdash; just fixed the square brackets and also changed the capitalization to be more consistent (and match other cheat sheets).

mintsoft commented:
@moollaza again, LGTM

moollaza commented:
@tagawa LGTM :+1:

tagawa commented:
Cool, thanks for the merge and reviews!
